packages/imperative/src/imperative/__tests__/Imperative.unit.test.ts

```diff
@@ -21,6 +21,7 @@ import * as yargs from "yargs";
 import { ImperativeError } from "../../error/src/ImperativeError";
 
 describe("Imperative", () => {
+    // eslint-disable-next-line deprecation/deprecation
     const mainModule = process.mainModule;
```

Comment:
Curious if there is a way to configure the deprecation/deprecation eslint plug-in to ignore all uses of `process.mainModule` 😋

Reply:
There is not. I checked.